### PR TITLE
More robust dev server configuration for demoing

### PR DESCRIPTION
**Pull request by banesullivan**
Reimplement/duplicate of https://github.com/ResonantGeoData/ResonantGeoData/pull/334

### CODE REVIEW

**Comment by aashish24** [comment]
Is this still relevant? Looks like this has not been updated since last 4 months?

**Comment by banesullivan** [comment]
Yes, this is relevant for any local demos. This draft PR is simply for reference and we've kept it open to keep it easy to find. I propose that we instead change this to documentation of some sort and place it upstream.

@zachmullen and @brianhelba, would this be relevant to include as documentation in the cookie-cutter? Or perhaps is there a better place for this sort of information (e.g. Notion)?

File: opengeo/settings.py [hunk 1]
```diff
@@ -110,6 +110,10 @@ class TestingConfiguration(OpenGeoMixin, TestingBaseConfiguration):
     CELERY_TASK_EAGER_PROPAGATES = True
 
 
+class NonDebugDevConfiguration(DevelopmentConfiguration):
+    DEBUG = True
```
Comment (banesullivan) [inline opengeo/settings.py:114]: @zachmullen, this is copied directly from your work in https://github.com/ResonantGeoData/ResonantGeoData/pull/334/files#diff-2c77f7552f3fa334decbc44137040144e567d15cbb3c076a6dd71f0ffbe8ba79R99-R100

This name is tripping me up... "NonDebug" implies `DEBUG = False` to me. Would you clarify this, please?

Further, the inherited `DevelopmentBaseConfiguration` sets `DEBUG = True` already: https://github.com/girder/django-composed-configuration/blob/a5cf7c68ef93ea764100ee1b8c7e48edbf0a7f81/composed_configuration/_configuration.py#L48

Comment (zachmullen) [reply opengeo/settings.py:114]: Oops, I think that's just a typo... it definitely should be `DEBUG = False`, good catch.

**Pull request closed without merging**